CONTRIBUTING.md

```diff
@@ -5,25 +5,31 @@ completely by volunteers in their spare time, so any contribution--no matter how
 small--is greatly appreciated. In this file, we'll outline how you can
 contribute to the project.
 
+If you have questions after reading this, you can join our Gitter channel via
+the badge at the top of `./README.md`.
+
 ## Overview
 
 Several of the main ways that you can contribute to the project include:
 
-* Updating [the website](https://github.com/xournalpp/xournalpp.github.io)
-  with additional information.
-* Submitting translation improvements via Crowdin (see below).
-* Contributing code changes with pull requests (PRs).
+* [Contributing website improvements](#contributing-website-improvements)
```

Comment:
I think the original line is better. This links to a section that has a link to the repository; why not just link directly?

Reply:
I thought the consistency would be nice, but I have changed it back :+1: